package ch11_exception;

public class ExceptionEx05 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		try {
			throw new Exception("=========예외 강제 발생==========");
		}catch(Exception e) {
			
		}
	}

}
